app/views/static_pages/home.html.erb

```diff
@@ -55,5 +87,10 @@
     <div class="row">
       <%= render partial: "shared/product" , collection: @products %>
     </div>
+    <div class="row mt-2 justify-content-center">
+      <div class="col-md-3">
+        <%== pagy_bootstrap_nav(@pagy) %>
```

Comment:
thừa dấu `=`

Reply:
không có dấu == là nó kh ra đâu à chị, nó sẽ không escape mấy cái giá trị mà không an toàn ở đây